esp/GBPlay/main/hardware/wifi.c

```diff
@@ -203,38 +234,43 @@ bool wifi_connect(const char* ssid, const char* password, bool force)
             // Connected to something else. Give up.
             // When the user chooses to connect, force = true.
             // When the connection manager tries to connect, force = false.
-            goto end;
```

Comment:
Logic is the same here. I just restructured it to be a bit cleaner. Turning on "hide whitespace" makes the diff easier to read.